File: modul 3/Tugas no 4 praktikum pemrograman 3/prak04.c
#include <stdio.h>
int main(){
    int N;
    scanf("%d", &N);

    if (N>=1 && N<10)
    {
        printf("satuan");
    }
    else if (N==0)
    {
        printf("nol");
    }
    else if (N>=11 && N<20)
    {
        printf("belasan");
    }
    else if (N>=20 && N<100 || N==10)
    {
        printf("puluhan");
    }
    else
    {
        printf("anda menginput melebihi limit bilangan");
    }
    
return 0;
    

}

File: modul 3/Tugas no 4 praktikum pemrograman 3/prak04.py
N = int(input())

if N >= 1 and N < 10:
    print("satuan")
elif N == 0:
    print("nol")
elif N >= 11 and N < 20:
    print("belasan")
elif N >= 20 and N < 100 or N == 10:
    print("puluhan")
else:
    print("Anda menginput melebihi limit bilangan")